devops/openshift/database/mongo-ha.yml

```diff
@@ -2,9 +2,9 @@
 kind: Template
 apiVersion: template.openshift.io/v1
 metadata:
-  name: forms-flow-db
+  name: forms
```

Comment:
I feel forms-db would be more suitable for mongo.

Reply:
we are not having multiple db for the same mongo instance, so named it that way.

Reply:
I also think that it would be good to have a suffix `-db`.

Reply:
:) are we calling aest db in postgres as aest-db? its similar to that. Its already a db. I can change it, but do we need to change? @andrepestana-aot @dheepak-aot. Last time when we created the forms-flow-db name we created it with respect to formsflow documentations, so followed it.